functions/calculateSGA.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Calculates Summary Group Average
+ * 
+ * @api {get} /api/calculateSGA Calculate SGA value
+ * @param {ita} Number Summed Individual Trait Average 
+ * @param {sailorCount} Number Count of Sailors in given group 
+ * @apiSuccess (200) {Object} raw `sga` and `roundedSGA` object
+ */ 
+exports.handler = async event => {
+    const givenITA = event.queryStringParameters.ita;
+    const givenSailorCount = event.queryStringParameters.sailorCount;
+    const calculatedSGA = givenITA / givenSailorCount;
+
+    return {
+        statusCode: 200,
+        body: JSON.stringify({ 
+            sga: calculatedSGA, 
+            sgaRounded: Number(Math.round(parseFloat(calculatedSGA + 'e2')) + 'e-2').toFixed(2)
```

Comment:
I did a bit more research and don't think my math was that sound so this is a bit more robust. I tested a few inputs and outputs and it looks like it's working as expected now.